src/components/modebar/buttons.js

```diff
@@ -55,17 +55,15 @@ modeBarButtons.toImage = {
             return;
         }
 
-        if(gd._snapshotInProgress) {
-            Lib.notifier('Snapshotting is still in progress - please hold', 'long');
```

Comment:
Nevermind. I see. It's here now: https://github.com/plotly/plotly.js/pull/446/files#diff-40d98cbce238527d9bffcf2ecd4efa9bR33